sys-block/nbdkit/nbdkit-1.32.4.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WANT_LIBTOOL=none
+inherit autotools
+
+DESCRIPTION="NBD server with stable plugin ABI and permissive license"
+HOMEPAGE="https://gitlab.com/nbdkit/nbdkit"
+SRC_URI="https://download.libguestfs.org/nbdkit/$(ver_cut 1-2)-stable/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE="curl ext2 nbd gnutls libguestfs libssh libvirt lzma selinux torrent zlib zstd"
```

Comment:
This is probably fine, but keep in mind that you don't need to expose a USE flag for everything:
- it's okay to enable something by default if it's very likely someone will want the functionality and/or common package
- does it make sense to toggle all of these on/off?
- it's ok to wait until someone asks if you don't see the point in something with a feature disabled

I suspect all of these are fine in this case, except for maybe nbd, but it's something to keep in mind.